src/comparator/message_comparator.py

```diff
@@ -68,10 +68,10 @@ def _compareNestedMessages(self, nested_msg_dict_original, nested_msg_dict_updat
         message_name_original = nested_msg_dict_original.keys()
         message_name_update = nested_msg_dict_update.keys()
 
-        for msgName in set(message_name_original) - set (message_name_update):
+        for msgName in set(message_name_original) - set(message_name_update):
```

Comment:
Why recreate the original and update sets each time? Just make `message_name_original` and `meessage_name_update` sets when you define them above.

Reply:
👍 Done, thank you!